src/frontend/src/flows/register/index.ts

```diff
@@ -89,6 +91,7 @@ export const registerFlow = async ({
   registrationAllowed: { isAllowed: boolean; allowedOrigins: string[] };
   pinAllowed: () => Promise<boolean>;
   uaParser: PreloadedUAParser;
+  connection: Connection;
```

Comment:
Wouldn't it be nicer to have the parameter `googleAllowed` directly here to avoid passing the whole "connection" object?